dist/components/endpoint/partials/httpForm.html

```diff
@@ -34,53 +34,58 @@ <h3 class="inline-header">Monitoring</h3>
             </div>
-          </div>
-          <div class="gf-form-group">
             <div class="gf-form-inline">
               <div class="gf-form">
-                <label class="gf-form-label width-9">Path</label>
-                <input class="gf-form-input max-width-30" type="text" ng-model="ctrl.checks['http'].settings.path" placeholder="www.example.com">
+                <label class="gf-form-label width-9">Port</label>
+                <input class="gf-form-input width-11" type="number" ng-model="ctrl.checks['http'].settings.port" min="1" max="65535">
               </div>
               <div class="gf-form">
-                <label class="gf-form-label width-4">Port</label>
-                <input class="gf-form-input max-width-5" type="number" ng-model="ctrl.checks['http'].settings.port" min="1" max="65535">
+                <label class="gf-form-label width-9">Timeout (s)</label>
+                <input class="gf-form-input width-11" type="number" ng-model="ctrl.checks['http'].settings.timeout" min="1" max="10">
+                <info-popover offset="0px -95px">The timeout, in seconds, is the amount of time our probe will wait for a response.</info-popover>
               </div>
             </div>
+          </div>
+          <div class="gf-form-group">
             <div class="gf-form-inline">
               <div class="gf-form">
+                <label class="gf-form-label width-9">Path</label>
+                <input class="gf-form-input width-16" type="text" ng-model="ctrl.checks['http'].settings.path" placeholder="www.example.com">
+              </div>
+              <div class="gf-form">
                 <label class="gf-form-label width-9">Method</label>
                 <div class="gf-form-select-wrapper">
                   <select class="gf-form-input gf-size-auto" ng-model="ctrl.checks['http'].settings.method" ng-options="m for m in ['GET', 'PUT', 'POST', 'DELETE', 'HEAD', 'PATCH']"></select>
                 </div>
               </div>
+            </div>
+            <div class="gf-form-inline">
+              <div class="gf-form">
+                <label class="gf-form-label width-9">Content Match</label>
+                <input class="gf-form-input width-16" type="text" ng-model="ctrl.checks['http'].settings.expectRegex">
+              </div>
               <div class="gf-form">
-                <label class="gf-form-label width-10">Content Match</label>
-                <input class="gf-form-input max-width-21" type="text" ng-model="ctrl.checks['http'].settings.expectRegex">
+                <label class="gf-form-label width-9">Download Limit</label>
+                <input class="gf-form-input width-11" type="text" ng-model="ctrl.checks['http'].settings.downloadLimit">
+                <info-popover>The maximum amount of data the probe will attempt to download (in bytes, max {{ctrl.quotas.downloadLimit.limit || '102400'}}).</info-popover>
```

Comment:
We could, but the issue is that currently the limits/quotas are reported by the API as a number of bytes, so users won't ever see it anyway because the API will report the quota in bytes.  We can add code in the frontend here to convert to/from the kb notation if we want, but then we'd also want to do the conversion in the error messages int he API too.